contracts/src/v0.8/keystone/test/CapabilityRegistry_GetCapabilityIds.t.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
```

Comment:
nit: file name inconsistent with the name of the test and the other file

Reply:
👍 will fix it in a follow-up to avoid merge conflict.